uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_absence-work-patterns.scss

```diff
@@ -19,3 +19,7 @@
 .chr_absence-work-pattern__work-pattern-listing {
   display: inline-block;
 }
+
+.chr_absence-work-pattern__alert {
+  margin-bottom: 0 !important;
```

Comment:
Because `#bootstrap-theme .alert` and `.crm-container .alert` has more priority and both of them has `margin-bottom: 20px;` set.